src/routes/adminRoutes.js

```diff
@@ -5,13 +5,24 @@ const adminController = require('../controllers/adminController');
 
 // Middleware for Basic Authentication
 router.use(
+
   expressBasicAuth({
     users: { [process.env.ADMIN_USERNAME]: process.env.ADMIN_PASSWORD },
     challenge: true, // Sends authentication challenge if credentials are not provided
     realm: 'admin site', // Authentication realm for the challenge
-  })
+  }) /* #swagger.tags = ['Admin'] 
+  #swagger.summary = 'Admin Login'
+  #swagger.description = 'Authenticate as an admin to access the admin page.'
+
+  #swagger.responses[200] = {
+    description: 'Admin authenticated successfully.',
+  }
+  #swagger.responses[401] = {
+    description: 'Authentication failed.',
+  } */
```

Comment:
I don't think swagger-autogen reads any comments on middleware. Or at least, I'm not seeing any of this show up in the docs :shrug: